### ISSUE TITLE
broken link to pdf

### ISSUE THREAD

**Issue by stijnvanhoey** [adokter/bioRad#95]
https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf is referred in the package documentation (on a few places), but not resolving. Any update of the link?

Following files the file is referred to:
```
./R/param.R:#' Table 16 in the \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM specification}.
./R/pvol.R:#' Table 16 in the \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM specification}.
./R/calculate_vp.R:#' \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM}
./R/bioRad.R:#' \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM}
./R/plot.scan.R:#' Table 16 in the \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM specification}.
./R/map.R:#' \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM specification}.
./R/plot.ppi.R:#' Table 16 in the \href{https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014_O4_ODIM_H5-v2.2.pdf}{ODIM specification}.
```

**Comment by adokter** [comment]
odd, somehow the underscores have changed into dashes, this is the correct link:
https://github.com/adokter/vol2bird/blob/master/doc/OPERA2014-O4-ODIM-H5-v2.2.pdf

**Comment by adokter** [comment]
we can also rename the file back to the dashed form, either way

**Comment by adokter** [comment]
fixed by commit adokter/vol2bird@94c1315688d677cb523fa38d3173d3f4a46b0db0
